Validation/CTPPS/python/ctppsHepMCDistributionPlotter_cff.py

```diff
@@ -0,0 +1,8 @@
+from Validation.CTPPS.CTPPSHepMCDistributionPlotter_cfi import *
+
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(CTPPSHepMCDistributionPlotter, useNewLHCInfo = True)
```

Comment:
> @makortel Using `run3_common` was decided on a [AlCaDB meeting](https://indico.cern.ch/event/1308035/) with Tamas Vami and Francesco Brivio, where we explicitly discussed which modifier to use. `run3_common` is also used in a already closed PR #42515

@Glitchmin @tvami @francescobrivio could you please remind me which was the rationale of using `run3_common` instead of `ctpps_2022`? `run3_common` is built including `ctpps_2022` nonetheless, therefore the final effect should be the same. Using `run3_common` could be preferred in case a possible `ctpps_2024` (for example) will fully replace `ctpps_2022` (as it was the case for `ctpps_2022` that fully replaced previous `ctpps_2018` for run3): of course, in that case for future runN eras one would need to customize that ctpps_2024 explicitly, while by customizing `run3_common` instead this setting will be propagated till all future runs, including Phase2 ones.

Reply:
@perrotta the point was that  `LHCInfoPer*` records are used in Run-3, and we don't plan to go back to Run-2/Run-1 with them (this is usually not the case when a new record is introduced). So putting that older PR under `run3_common` was very natural. Not sure what the (far) future plans (for a rereco) are for 900 GeV runs in 2021, but the `LHCInfoPer*` records should be populated there too, so using `ctpps_2022` might be misleading there. Otherwise probably `ctpps_2022` is good too.

Reply:
@tvami `ctpps_2022` is actually used to define the whole Run3 era, see [Era_Run3_cff.py](https://github.com/cms-sw/cmssw/blob/a35710c1d7e8d6d42cfe235bf92bdd5d57f573b7/Configuration/Eras/python/Era_Run3_cff.py): even in case of a possible rerun of 2021 data, that modifier will be picked by that era, in spite of having "2022" in its name. I agree that it would have been better to name `ctpps_2022` something like `run3_ctpps` since the beginning (as for the other subdetectors and subpackages): but here we are.

Therefore, for the modifications themselves using `ctpps_2022` or `run3_common` is completely equivalent. I tend to agree with @makortel that the "`run3_common` should not be used for subdetector-specific customizations". Beside that, I have not a strong opinion on that, and I wouldn't consider it a showstopper, unless @cms-sw/core-l2 people keep suggesting to enforce is.

Reply:
The design intent, that has also been mostly followed (but not everywhere, and not enforced), was that we create a Modifier for these "subdetector-specific" customizations, and then construct the Eras (technically ModifierChains) from these Modifiers kind of like building a house from lego bricks.

I realize now that the idea seems to be to change the default for Run3 and beyond to something, and then add another modifier to be able to switch the behavior back (to Run2 and earlier). If the customizations that are currently tied to `run3_common` would be done with another modifier, say `run3_ctpps_sim` (just for the example, please invent a better name), the `Run3_CTPPS_directSim` Era could be defined as
```py
Run3_CTPPS_directSim = Run3.copyAndExclude([run3_ctpps_sim])
```
To me this approach would be cleaner than customizing things back and forth.